gem/static/js/springster.js

```diff
@@ -32,17 +32,16 @@
     var content = document.getElementById("content-wrapper");
     var menuList = document.getElementById("nav-list");
 
-    //WHY IS THIS BG APPLIED
+    //REASONS FOR BACKGROUND
```

Comment:
with the Rubber-Band / Elastic Scroll effect on some smartphone browsers, this sets the background color when scrolling

Reply:
also reinitialises when changing from portrait to landscape

Reply:
![ezgif-4-db940d22f4](https://user-images.githubusercontent.com/9653693/36316964-7838e6a4-1344-11e8-9833-333fc2a52a95.gif)